sdk/src/main/java/io/dapr/actors/runtime/ActorRuntime.java

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+package io.dapr.actors.runtime;
+
+import java.util.HashMap;
+import java.util.Set;
+import java.util.function.Function;
+
+import io.dapr.actors.*;
+
+/**
+ * Contains methods to register actor types. Registering the types allows the runtime to create instances of the actor.
+ */
+public class ActorRuntime {
+    private static volatile ActorRuntime instance;
+    private static DaprAsyncClient daprAsyncClient;
+    private final String TraceType = "ActorRuntime";
+    private final HashMap<String, ActorManager> actorManagers;
+
+    private ActorRuntime() throws IllegalStateException{
+        if (instance != null) {
+            throw new IllegalStateException("ActorRuntime should only be constructed once");
+        }
+
+        this.actorManagers = new HashMap<String, ActorManager>();
+        daprAsyncClient = new DaprClientBuilder().buildAsyncClient();
+    }
+
+    /** TODO: maybe this should not be a singleton
```

Comment:
There will only be one ActorRuntime in user service.

Reply:
Removed comment.